src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { findCurent } from './helper';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const checkId = (currentTab, tab) => currentTab.id === tab.id;
```

Comment:
1. You can place this out of the component, to avoid recreation on each render
2. Please consider renaming this, as the function name is not informative. Possin]ble options: `isTabSelected`, etc